bcg6es-d1e110/cod-Zy765t_bcg6es-d1e110.xml

```diff
@@ -156,8 +156,8 @@
         <p xml:id="bcg6es-d1e110-d1e402">
           <lb ed="#P" n="23"/>§. Cognitis etiam Pseudo, debent Praelati praecipere Sub¬ 
           <lb ed="#P" n="24"/>lectis <!-- subiectis?--> suis, vt eos &amp; eorum doctrinam euitent dum fuerint ta¬ 
-          <lb ed="#P" n="25"/>les, iuxta praeceptum Apostoli <ref xml:id="bcg6es-d1e110-Rd1e504" corresp="#bcg6es-d1e110-Qd1e507">2. Timoth. 3.</ref> dicentis. <quote xml:id="bcg6es-d1e110-Qd1e507" source="http://scta.info/resource/IItim3_5" synch="9-11">Et hos 
-            <lb ed="#P" n="26"/>deuita</quote>. <ref xml:id="bcg6es-d1e110-Rd1e512">Glossa.</ref> <quote xml:id="bcg6es-d1e110-Qd1e515">Tales.</quote> Id est, <quote xml:id="bcg6es-d1e110-Qd1e520">dum fuerint tales.</quote> <!-- Issue aufmachen --> Item <ref xml:id="bcg6es-d1e110-Rd1e518">Rom.</ref>  
+          <lb ed="#P" n="25"/>les, iuxta praeceptum Apostoli <ref xml:id="bcg6es-d1e110-Rd1e504" target="#bcg6es-d1e110-Qd1e507">2. Timoth. 3.</ref> dicentis. <quote xml:id="bcg6es-d1e110-Qd1e507" source="http://scta.info/resource/IItim3_5" synch="9-11">Et hos 
```

Comment:
This was correct the way it was. Change `@target` to `@corresp`. 

Perhaps it will help to see that the `target` or `source` of this `<ref>` is really 2  timothy 3. But it is redundant to apply a `target` to the `ref` because that information is already captured in the `<quote>` below. So all we need to do is "tie" the `<ref>` to its "corresponding" `<quote>` and we have all the information we need.